nilrt_snac/_configs/_wireguard_config.py

```diff
@@ -25,13 +23,8 @@ def configure(self, args: argparse.Namespace) -> None:
         opkg_conf = _ConfigFile(OPKG_SNAC_CONF)
         ifplug_conf = _ConfigFile("/etc/ifplugd/ifplugd.conf")
         dry_run: bool = args.dry_run
-        subprocess.run(["wget", WIREGUARD_TOOLS_DEB, "-O", "./wireguard-tools.deb"], check=True)
-        if not opkg_conf.contains("arch amd64 15"):
-            opkg_conf.add("arch amd64 15\n")
-            # We need to save the file before installing the package so that amd64 is
-            # a valid architecture
-            opkg_conf.save(dry_run)
-        self._opkg_helper.install("./wireguard-tools.deb", force_reinstall=True)
+
+        self._opkg_helper.install("wireguard-tools", force_reinstall=True)
```

Comment:
Why do we need `force_reinstall`? Was it because we were using the debian package?

Reply:
Not sure. I'll try to remove that param.